### ISSUE TITLE
Design systems for developers missing image

### ISSUE THREAD

**Issue by jonniebigodes** [chromaui/learnstorybook.com#182]
While giving the finishing touches to the translation regarding the "Design Systems for Developers", when i've reached the "Document page" it seems that one image is missing. The markdown has the following content:
```markdown
![alt_text](/design-systems-for-developers/Feedback-wanted53.png)
```
Checking the `static` folder where the assets are stored, the image in question as you can see in the screenshot below:
![learn_design_systems_document_image_missing](https://user-images.githubusercontent.com/22988955/66688156-573a0c80-ec7d-11e9-9717-d1021e1f7f52.png)

On the left is the folder with the assets and on the right is the live version.

I've checked the repo and the image is also not there. 

I'm going to leave it as is and hope that soon the image is added.

Feel free to provide feedback

**Comment by tmeasday** [comment]
Thanks @jonniebigodes I guess I missed this!

**Comment by jonniebigodes** [comment]
@tmeasday no problem, glad to help out

**Comment by jonniebigodes** [comment]
Fixed with #184.
